### DISCUSSION TITLE
website content

### DISCUSSION THREAD

**Discussion by nsrmhjl** [nsrmhjl/ptDtSc#5] [General]
<div dir='rtl'>
سلام آقای @AlirezaDoustmohammadi 
خوبی؟
میشه بهم یاد بدی یکم مطلب توی صفحه اول وب سایتم بذارم؟
راستی آموزشی چی شد؟
</div>

**Comment by AlirezaDostmohammadi** [comment]
سلام دکتر جان. وقتتون بخیر. امیدوارم حالتون خوب باشه. سلامت باشید


آقای دکتر برای این کار می تونید از بخش زیر انجام بدید:

 Site administration> General 

در این بخش یه بخش blog داره که میتونید محتوا بنویسید و همچنین بخش Site home settings هست که اطلاعات سایت تون میتونید تغییر بدید.

https://docs.moodle.org/400/en/Change_your_front_page#:~:text=Click%20Site%20administration,from%20the%20two%20drop%20downs.

برای زیباتر شدن سایت تون هم میتونید از template های مودل استفاده کنید.

https://moodle.org/plugins/theme_adaptable

**Reply by nsrmhjl** [reply]
<div dir='rtl'>
ببخشید منظورم رو خوب نگفتم 
منظورم صفحه اول خود سایت mohajelresearch.net بود.

**Comment by AlirezaDostmohammadi** [comment]
اختیار دارید دکتر جان. چه محتوایی می خواید در صفحه اول قرار بدید؟ مثلا یه
چیزی شبیه صفحه اول آزمایشگاه ها باشه؟
مثل این؟
http://bioinf.modares.ac.ir/

**Reply by nsrmhjl** [reply]
<div dir='rtl'>
چقدر این بابا خودشو تحویل گرفته!
آره همین از سر منم زیاده.

**Comment by AlirezaDostmohammadi** [comment]
سلام ددکتر جان. خوبید؟
این قالب خوبی هست. ان شاالله حضوری درباره اش صحبت میکنیم.
https://wordpress.org/themes/astra/
